js/app.js

```diff
@@ -54,180 +51,59 @@ $(document).on('ready', function() {
-                            playerPos + '</div>');
-                          //}//for
-                          this.buildString();
-                          this.keyStroke(0);
-                    };//buildRow
+      $('#board').append('<div class="size '+this.playerColor+
+        '"><img class="avatar" id="' + this.playerColor + '" src=https://media.licdn.com/mpr/mpr/shrinknp_400_400/p/8/005/079/233/062b1d4.jpg> ' +
+        playerPos + '</div>');
+      this.buildString(); // create the series of divs for inside this row
+      this.handleKeyStroke(0);  // bind the keyup event for keyCode 47 + playerPos to this player, and note it's pressed 0 keys so far
+    };
+
+    // creates a  string for each space on the track row for this player
     this.buildString = function (){
-                            for(var t = 0; t < boardLength; t++){
-                              $('.'+this.playerColor).append('<div class="inline clear ' +
-                                this.playerLetter + '" id="' + letter + t + '">_______</div>');
-                                //<img id=a"'+letter+t+'" src="">');
-                            }//for
-                            //finish line
-                            $('.'+this.playerColor).append('<div class="inline">|</div>');
-                        };//buildString
-
-    //building the keypress automatiaclly
-    this.keyStroke = function key(pressNum){
-                        //console.log("The call is coming from inside the function!");
-                        //var counter = 0;
-                        //var not = keyPressNum;
-                        //var n = 0;
-                        //var pressNum = n;
-                        $(window).on("keyup", function handleKeypress(event) {
-                          if(event.keyCode === (47 + playerPos)){
-                            if(pressNum === (boardLength-1)){
-                              $('#' + letter + pressNum).text(".");
-                              $('#' + letter + (pressNum-1)).text("_______");
-                              console.log(pressNum);
-                              alert('player' + playerPos + ' has won!');
-                              //$('#board').remove();
-                              //$('.inline').remove();
-                              //resetKeypress(countPlayers);
-                              //keyPressNum = 0;
-                              $('.clear').text("_______");
+        for(var t = 0; t < boardLength; t++){
+          $('.'+this.playerColor).append('<div class="inline clear ' +
+            this.playerLetter + '" id="' + letter + t + '">_______</div>');
+        }
+        //finish line
+        $('.'+this.playerColor).append('<div class="inline">|</div>');
+    };
+
+    //binding the keypress automatically
+    this.handleKeyStroke = function (){
+      var that = this;  // save scope
```

Comment:
You'll see that=this to save context often in JavaScript.  Right now, it's because I want to be able to reference the instance variable pressNum from within the event handler that starts on line 86.  If I forgot to use `that` to save `this`, doing `this.pressNum` would be looking for a `pressNum` inside the `event` instead of inside the player instance.